string ACTION_TYPE_MOVE=move
string ACTION_TYPE_ATTACH=attach
string ACTION_TYPE_DETACH=detach
string ACTION_TYPE_ROTATE=rotate
string ACTION_TYPE_CONNECT=connect
string ACTION_TYPE_REQUEST=request
string ACTION_TYPE_SUBMIT=submit

string id
string action_type
diagnostic_msgs/KeyValue[] params
uint64 target_deadline

string agent